shopinvader_auth_api_key/components/service_context_provider.py

```diff
@@ -12,10 +12,10 @@
 _logger = logging.getLogger(__name__)
 
 
-class AuthApiKeyServiceContextProvider(Component):
+class ShopinvaderAuthApiKeyServiceContextProvider(Component):
     _name = "auth_api_key.shopinvader.service.context.provider"
     _inherit = "shopinvader.service.context.provider"
-    _usage = "auth_api_key_component_context_provider"
+    _usage = "shopinvader_auth_api_key_component_context_provider"
```

Comment:
"component" seems redundant here
```suggestion
    _usage = "shopinvader_auth_api_key_context_provider"
```

Reply:
Good idea